src/main/java/org/folio/circulation/resources/ExpiredSessionProcessingResource.java

```diff
@@ -74,6 +78,7 @@ private void process(RoutingContext routingContext) {
   }
 
   private CompletableFuture<Result<ZonedDateTime>> defineExpiredTime(Integer timeout) {
+    log.debug("defineExpiredTime:: parameters timeout: {}", timeout);
```

Comment:
What about process() method? We logged it in ExpiredActualCostProcessingResource.java